nextflow

```diff
@@ -15,7 +15,7 @@
 #  limitations under the License.
 
 [[ "$NXF_DEBUG" == 'x' ]] && set -x
-NXF_VER=${NXF_VER:-'18.12.0-edge'}
+NXF_VER=${NXF_VER:-'19.1.0-SNAPSHOT'}
 NXF_ORG=${NXF_ORG:-'nextflow-io'}
```

Comment:
As before, these numbers are automatically updates, do not add this file to the commit.